letsencrypt/cli.py

```diff
@@ -669,7 +682,12 @@ def determine_verb(self):
 
         for i, token in enumerate(self.args):
             if token in self.VERBS:
-                self.verb = token
+                verb = token
+                if verb == "certonly":
+                    verb = "auth"
```

Comment:
I'm fine with that plan in the medium term, I just wanted this to be a minimal code change before launch rather than a far-reaching one.